### DISCUSSION TITLE
endless waiting for runner

### DISCUSSION THREAD

**Discussion by JeWaVe** [community/community#129079] [Actions]
### Select Topic Area

Bug

### Body

In an attempt to speed up my CI (private repo of my organization), I built a dockerimage with everything preinstalled that I pushed on dockerhub (ubuntu24.04 derived)

image is dockeraltimesh/build:ubuntu24.04 and the registry on dockerhub is public

However my CI waits for a runner like forever. 

I have searched for similar issues, but OS (ubuntu 24) is supported by github and I didn't misspell anything (AFAIK) 

![image](https://github.com/community/community/assets/11440561/f692b246-c1ee-497c-a153-02bfae618eea)

![image](https://github.com/community/community/assets/11440561/91b7083c-2070-4c53-b3a7-43bf67993361)

**Comment by JeWaVe** [comment]
well, like really stuck 
![image](https://github.com/community/community/assets/11440561/c4b41065-567a-40a4-941b-d05048407cc4)

**Comment by mickeygousset** [comment]
https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job

I think what you want to be doing is targeting "ubuntu-latest" to use a hosted runner, and then specifying your job to use the container.

the "runs-on" command is used to specify a label or series of labels on a runner.  How you are referencing it will not work.

**Comment by JeWaVe** [comment]
Well it's way better. Sorry for the misreading of documentation
